docs/sphinx_setup/_static/html/banner_data.json

```diff
@@ -1,6 +1,6 @@
 {
-    "p": "",
+    "p": "This page is a nightly version. It may be incomplete or faulty i both content and functionality. <a href='https://docs.openvino.ai/'>Go to the most recent official documentation version, 2024.</a>",
```

Comment:
```suggestion
    "p": "This page is a nightly version. It may be incomplete or faulty in both content and functionality. <a href='https://docs.openvino.ai/'>Go to the most recent official documentation version, 2024.</a>",
```